packages/squiggle-lang/__tests__/ast/parse_test.ts

```diff
@@ -161,6 +162,10 @@ describe("Peggy parse", () => {
       "{a: 1, b: 2}",
       "(Program (Record (KeyValue 'a' 1) (KeyValue 'b' 2)))"
     );
+    testParse(
+      "{a, b, }",
+      "(Program (Record (KeyValue 'a' :a) (KeyValue 'b' :b)))"
+    );
```

Comment:
I might suggest adding some more tests. At least ``{a,b}``. 
(Not a major issue though)